### ISSUE TITLE
Pagination component: underlying HTML changes for a11y

### ISSUE THREAD

**Issue by jbadan** [SAP/fundamental#1352]
Pagination component: https://sap.github.io/fundamental/components/pagination.html

 `<a>` tags should be `<button>`
